lwgenerate/_examples/aws/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lacework/go-sdk/lwgenerate"
+)
+
+func main() {
+	hcl, err := lwgenerate.NewAwsTerraform("us-east-1", true, true).Generate()
```

Comment:
Can we add more examples with some of the more complicated use cases and comments.

Reply:
👍